### PR TITLE
feat: support kubeconfig-path for release-cli

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @caitong93 @zoumo @kdada @li-ang 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/release-cli/create.go [hunk 1]
```diff
@@ -57,16 +64,31 @@ func runCreate(cmd *cobra.Command, args []string) {
...
+		clientset, err = kubernetes.NewForConfig(&rest.Config{
+			Host:        createOptions.Server,
+			BearerToken: createOptions.BearerToken,
+			TLSClientConfig: rest.TLSClientConfig{
+				Insecure: true,
+			},
+		})
+		if err != nil {
+			glog.Fatalln(err)
+		}
```
Comment (zoumo) [inline cmd/release-cli/create.go]: 上面这段代码重复 4 次，能否抽离出来做一个公共函数，参考 compass-admin

**Comment by zhujian7** [comment]
@zoumo @li-ang PTAL

File: cmd/release-cli/create.go (hunk 1 above)
Comment (zhujian7) [reply cmd/release-cli/create.go]: have fixed

File: cmd/release-cli/create.go [hunk 2]
```diff
@@ -34,8 +37,12 @@ var create = &cobra.Command{
 }
 
 func runCreate(cmd *cobra.Command, args []string) {
-	if createOptions.Server == "" || createOptions.BearerToken == "" {
-		glog.Fatalln("--server and --bearer-token must be set")
+	if createOptions.Server == "" {
+		glog.Fatalln("--server must be set")
+	}
+
+	if createOptions.BearerToken == "" && createOptions.KubeconfigPath == "" {
```
[the remark this answers is not in the document]
Comment (zhujian7) [reply cmd/release-cli/create.go]: have fixed

**Pull request merged**